docs/guides/deployment/digitalocean.rst

```diff
@@ -0,0 +1,105 @@
+.. _ref_guide_deployment_digitalocean:
+
+===============
+On DigitalOcean
+===============
+
+In this guide we show how to deploy EdgeDB to DigitalOcean using a DigitalOcean
+managed PostgreSQL database as the backend.
+
+
+Prerequisites
+=============
+
+* DigitalOcean account
+* ``doctl`` CLI (`install <doclt-install_>`_)
+
+.. _doctl-install: https://docs.digitalocean.com/reference/doctl/how-to/install/
+
+
+Create a managed PostgreSQL instance
+====================================
+
+Note the URI that is printed in the output from this command. It will be used
+in a later step.
+
+.. code-block:: bash
+
+   doctl databases create edgedb-postgres \
+       --engine pg \
+       --version 12 \
+       --size db-s-1vcpu-1gb \
+       --num-nodes 1 \
+       --region sfo2
+
+
+Provision a droplet
+===================
+
+Replace ``$SSH_KEY_IDS`` with the ids for the ssh keys you want to ssh into the
+new droplet with seperating multiple values with a comma. You can list your
+keys with ``doctl compute ssh-key list``.  If you don't have any ssh keys in
+your DigitalOcean account you can follow `this guide <upload-ssh-keys_>`_ to do
+that now.
+
+Note the PublicIPv4 that is printed in the output from this command. It will be
+used in the next step.
+
+.. _upload-ssh-keys: https://docs.digitalocean.com/products/droplets/how-to/add-ssh-keys/to-account/
+
+.. code-block:: bash
+
+   doctl compute droplet create edgedb \
+       --image docker-20-04 \
+       --region sfo3 \
+       --size s-2vcpu-4gb \
+       --ssh-keys $SSH_KEY_IDS \
+       --wait
+
+
+Start EdgeDB
+============
+
+Replace ``$IP`` and ``$URI`` with the values from the previous steps. And
+replace ``$PASSWORD`` with a strong password for the EdgeDB administrator user.
+
+.. code-block:: bash
+
+   ssh root@$IP "
+       docker run \
+           --detach \
+           --restart always \
+           --name edgedb \
+           --publish 5656:5656 \
+           --volume /var/lib/edgedb/data:/var/lib/edgedb/data \
```

Comment:
The data volume is not necessary when `SERVER_BACKEND_DSN` is used.

Reply:
What about persisting the generated TLS certificate?

Reply:
Ideally, it needs to be extracted and stored as a secret like we do for Fly.

Reply:
I don't think that digitalocean has a builtin secrets management solution. I couldn't find one. We would have to choose a secrets management solution like Vault or something and send the reader off to complete some other tutorial before continuing with ours.

Reply:
Or we could change the tutorial to use Kubernetes which has secrets management.

Reply:
Hm, I actually think we should change this to use our pre-built image instead.